test/e2e/tests/settings/account-token-list.spec.js

```diff
@@ -56,11 +56,23 @@ describe('Settings', function () {
           text: 'General',
           tag: 'div',
         });
-        await driver.clickElement({ text: 'Fiat', tag: 'label' });
+        await driver.clickElement('.show-native-token-as-main-balance');
+        // We now need to enable "Show fiat on testnet" if we are using testnets (and since our custom
+        // network during test is using a testnet chain ID, it will be considered as a test network)
```

Comment:
Would it be better to add this to the fixture builder?

```
.withPreferencesController({
    preferences: {
        showFiatInTestnets: true
    }
})
```

Reply:
I can refactor that! 🙏  I think the purpose of this test is to make sure tokenList and accountList item match; so we can skip the toggling of showFiat in settings and have them correctly set in fixtures! [b101a24](https://github.com/MetaMask/metamask-extension/pull/27097/commits/b101a242238ee0a867425ebe93653e4e9982e0b9)